.github/workflows/main.yml

```
@@ -0,0 +1,59 @@
+env:
+  RUBY_VERSION: 2.3.7
+  FLOW_TOKEN: testflow
+
+name: Code Quality
+
+on: [push]
+
+jobs:
+  rspec-test:
+    name: RSpec
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:11.6
+        ports: ["5432:5432"]
+        env:
+          POSTGRES_USER: ${{ secrets.POSTGRES_USER }}
+          POSTGRES_PASSWORD: ${{ secrets.POSTGRES_PASSWORD }}
+          POSTGRES_DB: ${{ secrets.POSTGRES_DB }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v1  
```

We could try a modern version as well:

```suggestion
      - uses: actions/cache@v2.1.3
```